appbars/src/main/res/values/styles-topbars.xml

```diff
@@ -57,20 +57,24 @@
     <eat-comment />
 
     <style name="Widget.Vitamin.TopBar.Search" parent="@style/Widget.Vitamin.TopBar">
-        <item name="contentInsetStart">@dimen/vtmn_appbars_topbar_search_content_inset_start</item>
         <item name="materialThemeOverlay">@style/ThemeOverlay.Vitamin.TopBar.Search</item>
     </style>
 
     <style name="ThemeOverlay.Vitamin.TopBar.Search" parent="">
-        <item name="colorOnPrimary">?attr/vtmnContentColorTertiary</item>
+        <item name="colorOnPrimary">?attr/vtmnContentColorPrimary</item>
+        <item name="android:textColorHint">?attr/vtmnContentColorTertiary</item>
     </style>
 
     <style name="Widget.Vitamin.SearchView" parent="@style/Widget.AppCompat.SearchView">
         <item name="closeIcon">@drawable/ic_vtmn_close_line</item>
-        <item name="searchIcon">@drawable/ic_vtmn_search_line</item>
```

Comment:
Does it mean that  `ic_vtmn_search_line` is no more the drawable used as icon at the left of the top app bar?

Reply:
The searchView usage in Vitamin is: 
if you click the `search` icon in the topAppBar option, it opens a new screen with the searchView deployed.
In this case (and this is right now the only case), the search icon should not be visible in the searchView as there is already an icon at the left: the back icon. You can see the behavior in the screenshots